llvm/lib/Target/AArch64/AArch64Features.td

```diff
@@ -369,9 +369,12 @@ def FeatureSVE2 : ExtensionWithMArch<"sve2", "SVE2", "FEAT_SVE2",
   "Enable Scalable Vector Extension 2 (SVE2) instructions",
   [FeatureSVE, FeatureUseScalarIncVL]>;
 
-def FeatureSVE2AES : ExtensionWithMArch<"sve2-aes", "SVE2AES",
+def FeatureSVEAES : ExtensionWithMArch<"sve-aes", "SVEAES",
   "FEAT_SVE_AES, FEAT_SVE_PMULL128",
-  "Enable AES SVE2 instructions", [FeatureSVE2, FeatureAES]>;
+  "Enable SVE AES and 128-bit PMULL instructions", [FeatureAES]>;
```

Comment:
I think what you mean is:
"Enable SVE AES and SVE PMULL instructions", am I right? or you actually meant Neon PMULL?

Reply:
I'm referring to the quadword variants of [PMULLB/PMULLT](https://developer.arm.com/documentation/ddi0602/2024-09/SVE-Instructions/PMULLB--Polynomial-multiply-long--bottom--) here. Perhaps I should change this to "Enable SVE AES and 128-bit elment SVE PMULL instructions".

Reply:
I think that would be more clear, from the original text I think it looked more like the SVE relates only to the AES, not to PMULL, and PMULL has variants for NEON and SVE.